src/parser/parser.yy

```diff
@@ -2843,22 +2869,46 @@ create_space_sentence
         auto sentence = new CreateSpaceSentence($4, $3);
         $$ = sentence;
     }
+    | KW_CREATE KW_SPACE opt_if_not_exists name_label KW_COMMENT ASSIGN STRING {
+        auto sentence = new CreateSpaceSentence($4, $3);
+        sentence->setComment($7);
+        $$ = sentence;
+    }
     | KW_CREATE KW_SPACE opt_if_not_exists name_label KW_ON name_label {
         auto sentence = new CreateSpaceSentence($4, $3);
         sentence->setGroupName($6);
         $$ = sentence;
     }
+    | KW_CREATE KW_SPACE opt_if_not_exists name_label KW_ON name_label KW_COMMENT ASSIGN STRING {
```

Comment:
Need we to add a comma between group and comment? @CPWstatic @dutor any suggestion?

Reply:
I think we should add comma between `comment` and successor field.And don't need add after group name.